mep/books/templates/books/work_detail.html

```diff
@@ -20,21 +20,19 @@
 </nav>
 <section class="details">
     {% include 'books/snippets/work_result.html' %}
-</section>
+
 {% if work.edition_set.exists %}
-<section class="volumes-issues">
+<div class="volumes-issues">
     <h2>Volume/Issue</h2>
```

Comment:
the `work_result` snippet that gets rendered above inside this `section` does not contain any headings, so `Volume/Issue` is the heading that will be used to title the entire details `section`. that seems like not what we want. I think we could either add an aria-label to the entire `section` so that it doesn't fall back to this `h2` (maybe just `section aria-label="details"`), or use something other than a heading here and style it accordingly (less preferable, imo).